tests/integration/create_token.py

```diff
@@ -163,5 +163,18 @@ def create_token_with_schema_url(self, schema_name, schema_url, **extra_payload)
 
         return self.generate_token(payload_vars)
 
+    def create_token_v2_with_none_language_code(
+        self, schema_name, theme="default", **extra_payload
+    ):
+        payload_for_theme = (
```

Comment:
Appreciate this mirrors `create_token_v2` but I think for this one theres no need to have the "theme" parameter and the conditional logic based on theme

Reply:
I had it that way originally, I opted to keep it for extensibility in any future tests

Reply:
Removed 👍